doc/users/prev_whats_new/whats_new_3.10.0.rst

```diff
@@ -292,6 +292,62 @@ the ``set_data`` method, enabling e.g. resampling
     fig.savefig("after.png")
 
 
+Added ``matplotlib.colorizer.Colorizer`` as a container for ``norm`` and ``cmap``
+-------------------------------------------------------------------------------
+
+ `matplotlib.colorizer.Colorizer` encapsulates the data-to-color pipeline. It makes reuse of colormapping easier, e.g. across multiple images. Plotting methods that support ``norm`` and ``cmap`` keyword arguments now also accept a ``colorizer`` keyword argument.
+
+In the following example the norm and cmap are changed on multiple plots simultaneously:
+
+
+.. plot::
+    :include-source: true
+    :alt: Example use of a matplotlib.colorizer.Colorizer object
+
+    import matplotlib.pyplot as plt
+    import matplotlib as mpl
+    import numpy as np
+
+    x = np.linspace(-2, 2, 50)[np.newaxis, :]
+    y = np.linspace(-2, 2, 50)[:, np.newaxis]
+    im_0 = 1 * np.exp( - (x**2 + y**2 - x * y))
+    im_1 = 2 * np.exp( - (x**2 + y**2 + x * y))
+
+    colorizer = mpl.colorizer.Colorizer()
+    fig, axes = plt.subplots(1, 2, figsize=(6, 2))
+    cim_0 = axes[0].imshow(im_0, colorizer=colorizer)
+    fig.colorbar(cim_0)
+    cim_1 = axes[1].imshow(im_1, colorizer=colorizer)
+    fig.colorbar(cim_1)
+
+    colorizer.vmin = 0.5
+    colorizer.vmax = 2
+    colorizer.cmap = 'RdBu'
+
+All plotting methods that use a data-to-color pipeline now create a colorizer object if one is not provided. This can be re-used in by subsequent artists such that they will share a single data-to-color pipeline:
```

Comment:
Also wrap.
```suggestion
All plotting methods that use a data-to-color pipeline now create a colorizer object if one is not provided. This can be re-used by subsequent artists such that they will share a single data-to-color pipeline:
```